code/modules/core_implant/cruciform/rituals/machinery.dm

```diff
@@ -74,6 +74,73 @@
 	forge.produce()
 	return TRUE
 
+//Airlocks
+
+/datum/ritual/cruciform/machines/lock_door
+	name = "Activate door"
+	phrase = "Inlaqueatus"
+	desc = "Commands nearby door to be locked or unlocked."
+
+/datum/ritual/cruciform/machines/lock_door/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/C)
+	var/list/OBJS = get_front(user)
+
+	var/obj/machinery/door/holy/door = locate(/obj/machinery/door/holy) in OBJS
+
+	if(!door)
+		fail("You fail to find a compatible door here.", user, C)
+		return FALSE
+
+	if(door.stat & (BROKEN))
+		fail("[door] is off.", user, C)
+		return FALSE
+
+	door.locked ? door.unlock() : door.lock()
+	return TRUE
+
+/datum/ritual/cruciform/machines/repair_door
+	name = "Repair door"
+	phrase = "Redde quod periit"
+	desc = "Repairs nearby door at the cost of biomatter."
+
+/datum/ritual/cruciform/machines/repair_door/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/C)
+	var/list/OBJS = get_front(user)
```

Comment:
Don't make all caps variables please.

Reply:
Copycatting from existing litanies. Why not apply same requirements to all coders?